kornia/geometry/transform/affwarp.py

```diff
@@ -581,12 +581,14 @@ def resize(
 
     if antialias:
         # First, we have to determine sigma
-        sigmas = (max(factors[0], 1.0), max(factors[1], 1.0))
+        # Taken from skimage: https://github.com/scikit-image/scikit-image/blob/v0.19.2/skimage/transform/_warps.py#L171
+        sigmas = (max((factors[0] - 1.0) / 2.0, 0.001),
+                  max((factors[1] - 1.0) / 2.0, 0.001))
```

Comment:
With 0.001 instead of 0. As scikt suggests  ?

Reply:
There is no sense is sigma 0

Reply:
To apply "by default" for upsampling also does not make sense :)